README.md

```diff
@@ -1,28 +1,40 @@
-# Team Open workgroups
-
-**[![Workgroups](badge.svg)](workgroups)**
-
-Workgroups are responsible for particular tasks within our community.
-By letting them be autonomous and responsible for their respective tasks there are two main benefits:
-
-- Authority can be more decentralized.
-- Contributors can more easily help with the particular tasks they're interested in.
-
-### Joining/leaving a workgroup
-
-Workgroups are open to anyone. If you're interested, you can join by creating a new issue using
-the "Join workgroups" issue template.
-
-Likewise you can leave whenever you like with the "Leave workgroups" issue template.
-
-### Roles within a workgroup
-
-As a first time member you'll likely be assigned a "trial" role first.
-Within a workgroup there can be different roles. Each workgroup can define these for themselves.
-
-### Proposing new workgroups
-
-The Governance and Guidance workgroup reviews proposals for new workgroups.
-There is no formalized process for this, so feel free to create an issue or reach out to one of the
-workgroup members.
-
+<!-- GENERATED FILE, DON'T EDIT -->
+# Team Open - Workgroup list
+
+[![Get involved](get-involved.svg)](https://github.com/teamopen-dev/workgroups/issues/new?template=wg-join-leave.md&title=Would%20like%20to%20join)
+&nbsp; <sup>[More info on workgroups](ABOUT.md)</sup>
+
+Title | Summary | Members
+-|-|-:
+[Organization][wg-organization]|Organization level day-to-day tasks.|2
+[Outreach][wg-outreach]|Reaching out to where Team Open may help.|2
+[Lifeguard][wg-lifeguard]|Development of the Lifeguard project itself.|2
+[Lifeguard Devs][wg-lifeguard-devs]|People who maintain OSS in the Lifeguard programe.|2
+[Media][wg-media]|(Social) media, publication and communication tasks.|2
+
+# List of roles
+
+Workgroup | Role | Summary
+-|-|-
+[Organization][wg-organization]|`credentials`|People who handle login details / credentials for any part of the organization and share it with the appropriate other workgroups and members.
+[Organization][wg-organization]|`workgroups`|People who review applications for and generally facilitate the operation of workgroups.
```

Comment:
People who review membership applications and generally facilitate the operation of workgroups.

Reply:
Actually `membership applications` isn't right. It's reviewing proposals for new workgroups.

Reply:
People who review new workgroup proposals and generally facilitate the operation of workgroups.?

Reply:
Or just leave out that detail and keep it to:
People who facilitate the operation of workgroups.